app/views/help/cookie_settings.html.erb

```diff
@@ -116,4 +119,10 @@
       text: t('cookies.save_changes')
     } %>
   </form>
+
+  <%= render "govuk_publishing_components/components/govspeak", {
```

Comment:
I'd be tempted not to render this using the govspeak component. Instead, we can use the heading component, e.g:

```
<%= render "govuk_publishing_components/components/heading", {
    text: "Government services",
    padding: true,
    heading_level: 2
  } %>
```

and keep the paragraph as it currently is.